sdk/search/Azure.Search.Documents/src/Options/SearchOptions.cs

```diff
@@ -428,6 +428,7 @@ private static void Copy(SearchOptions source, SearchOptions destination)
             destination.QuerySpeller = source.QuerySpeller;
             destination.SemanticSearch = source.SemanticSearch;
             destination.VectorSearch = source.VectorSearch;
+            destination.HybridSearch  = source.HybridSearch;
```

Comment:
Could you please update [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/tests/DocumentOperations/SearchTests.cs#L1006) test case and add `HybridSearch` property for verification? Thank you!

Reply:
@ericvdberge Added the unit test